pkgs/by-name/gn/gnome-connections/package.nix

```diff
@@ -20,13 +20,16 @@
   gnome,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "gnome-connections";
-  version = "47.0";
+  version = "47.2.1";
```

Comment:
Release is already on the download server: https://download.gnome.org/sources/gnome-connections/47/